saskatoon/member/urls.py

```diff
@@ -8,9 +8,9 @@
          views.PersonCreateView.as_view(),
          name='person-create'),
 
-    path('beneficiary/create/',
+    path('organization/create/',
```

Comment:
Before changing the actual routes we need to make sure docs don't mention them anywhere, in which case think it could be okay just changing the reference name